install/yaml/02-open-match.yaml

```diff
@@ -204,7 +204,7 @@ spec:
     spec:
       containers:
       - name: om-backend
-        image: "gcr.io/open-match-public-images/openmatch-backendapi:0.4.0"
+        image: "gcr.io/jeremyedwards-gaming-dev/openmatch-backendapi:0.4.0-fc52ef6"
```

Comment:
Is there a specific reason that we are not using the public images for all open match components?